src/vm/database/sqlite.rs

```diff
@@ -49,12 +50,38 @@ impl ContractDatabaseConnection {
                             NO_PARAMS);
 
         contract_db.execute("CREATE TABLE IF NOT EXISTS simmed_block_table
-                      (simmed_block_height BLOB NOT NULL)",
+                      (block_height INTEGER PRIMARY KEY,
```

Comment:
Not required just yet, but you should eventually be able to pull all this information from the `BurnDB` database.